Nodejs/Product/Nodejs/NpmUI/NpmWorker.cs

```diff
@@ -103,6 +104,8 @@ private void Run()
                 return Enumerable.Empty<IPackage>();
             }
 
+            TelemetryHelper.LogSearchNpm(filterText);
```

Comment:
Likewise, logging what the user has typing in the search box may have PII issues.